src/routes/+page.js

```diff
@@ -51,6 +51,8 @@ frontPage = {
         background: "/assets/media/bg.svg",
         columns: false,
         divider: true,
+        boxed: true,
+        title: "Comparison",
```

Comment:
```suggestion
        title: "The Spyder advantage",
```

This is the title suggest by @CAM-Gerlach for this page and I think it's really good, so let's use it instead.